### ISSUE TITLE
train 

### ISSUE THREAD

**Issue by zzchuman** [cyrilhokage/DQN_CIL4SYS#1]
Hello, I find you are train the traffic light by rllib and sumo, And, I just want to run it. But, I can not find the main.py.
Can you speak english ? It sems like Freanch.
I can not understand your paper and ppt.

**Comment by zzchuman** [comment]
Maybe you can introduce you each folder function and how to use it.

**Comment by julienguegan** [comment]
If you just want to play and run some code you might first check the Flow library from Berkeley

**Comment by zzchuman** [comment]
I look you have use the ppo. each agent( a intersection) use a PPO ?
Right?
Does the adjacent intersection (agent) has the adjacent intersection status information?

**Comment by zzchuman** [comment]
> If you just want to play and run some code you might first check the Flow library from Berkeley

I have install  the flow.

**Comment by julienguegan** [comment]
Read the readme from https://github.com/flow-project/flow and they have also website and papier. And there is tutorials is their folder. It is better than our code, if it is really our code that you want I could explain more when I have time

**Comment by zzchuman** [comment]
Ok, I will go to read the flow. And can you solve my question? 

I can not find the main.py.

I look you have use the ppo. each agent( a intersection) use a PPO ?
Right?
Does the adjacent intersection (agent) has the adjacent intersection status information?

**Comment by julienguegan** [comment]
Yes the agent intersection has information about vehicles and intersections states

**Comment by zzchuman** [comment]
> Yes the agent intersection has information about vehicles and intersections states

I know, And Iwant to know whether the agent has another agent indormation？ 

I can not find you define the action and status, it was define in FLOW?

**Comment by julienguegan** [comment]
I think it is in notebook/traffic_light/IssyEnv.py and the function get_state() but note that there is only one reinforcement agent, we did not use a multi agent framework

**Comment by zzchuman** [comment]
Ok, thank you!  I will go to read the flow.  Thank you!
